gui/main_window.py

```diff
@@ -138,9 +138,10 @@ def __init__(self):
         super(MainWindow, self).__init__('statusDesktop_mainWindow')
         self.left_panel = LeftPanel()
 
-    def prepare(self) -> 'Window':
-        MockedKeycardController().wait_until_appears().hide()
-        return super().prepare()
```

Comment:
I think here we only need to comment MockedKeycardController().wait_until_appears().hide()
Because we need method prepare itself in many tests

Reply:
@Valentina1133 i was able to have all tests passed without this method, but sure, i will bring it back, sorry about that

Reply:
Thank you. Maybe because we use it in skipped tests. For example, I use it in keycard tests.

Reply:
done @Valentina1133 ! Good catch